src/convertToObject.js

```diff
@@ -12,7 +12,13 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').map(el => el.slice(3).split(': '))
```

Comment:
1. splice(3) -- someone may pass a string into this function which uses 4 spaces, or tabs. 

we should use trim() instead. 

2. same with ':  ' -> use ':' and use trim().